### DISCUSSION TITLE
Dreambooth tab greyed out, can't update other extensions without deleting the dreambooth

### DISCUSSION THREAD

**Discussion by ledeni00** [d8ahazard/sd_dreambooth_extension#981] [Q&A]
Hi,
Last night after the dreambooth extension update, the dreambooth tab got "greyed out" (cant use it) and i cant update other extensions without deleting the dreambooth ext.
Any suggestions? :)
If this helps:
Checking Dreambooth requirements...
[+] torch version 1.13.1+cu117 installed.
[+] torchvision version 0.14.1+cu117 installed.
[+] bitsandbytes version 0.35.4 installed.
[+] diffusers version 0.13.1 installed.
[+] transformers version 4.26.1 installed.
[+] xformers version 0.0.17.dev464 installed.
![Screenshot 2023-03-01 15 43 06](https://user-images.githubusercontent.com/122018478/222174420-10e2a5dc-1cf9-40a4-a3ef-602bffe1f9b7.png)

**Comment by Dasor92** [comment]
in the left section just create a model or select one.

**Reply by ledeni00** [reply]
> in the left section just create a model or select one.

Thank you.
My bad on that one. 
Haven't noticed the change in the layout.
The extensions update problem is still there. When i "check for updates" under the "update" column it says "unknown"

There's this when checking for updates:
cmdline: git fetch -v -- origin --dry-run
stderr: 'fatal: couldn't find remote ref --dry-run'

**Comment by Sh4d0wHunt3rX** [comment]
I confirm the issue as well. This is my first time I installed Dreambooth. But not possible to update any extensions now:

https://i.imgur.com/gucT2gu.png

Also, it doesn't show any of the models I have, so when I click a narrow tooltip opens 1 px hight as you can see in the image:

https://i.imgur.com/XmdkoRD.png

**Reply by Arilziem** [reply]
You need to first create a new dreambooth model in the "create tab". There you can choose what model you want to base the training on. After creating the model you should then see it in the dropdown in the screenshot.

About the updating issue, see my comment below.

**Comment by Arilziem** [comment]
Extension updating is broken due to a [security issue in GitPython](https://github.com/advisories/GHSA-hcpj-qp55-gfph), which prompted a breaking API change, which in turn broke the auto1111 webui update mechanism. See #982.

This should be sorted out soon-ish (fingers crossed). If you feel adventurous (or you know what you are doing), you can manually merge the PR linked in the issue. Until then, updating extensions will still work using git manually. The whole update mechanism is built on top of git after all.
